readhomer_atlas/library/models.py

```diff
@@ -352,23 +352,46 @@ class Token(models.Model):
         "Node", related_name="tokens", on_delete=models.CASCADE
     )
 
-    value = models.CharField(max_length=255)
-
+    value = models.CharField(
```

Comment:
@jtauber when you're able, please take a look at the help text strings I've added.

Much like we've traditionally used the `help_text` on model fields to serve as developer documentation (or presented those values on forms to users), I tried to capture my rough understanding of each of these properties.

If you have better help text in mind, I'm happy to bring it in 😉 

![image](https://user-images.githubusercontent.com/629062/80757842-6f2e8c00-8afa-11ea-866d-081f644825c4.png)